tmt/steps/prepare/install.py

```diff
@@ -261,13 +295,13 @@ def go(self, guest):
                     f"{packages}{yum_check}")
             else:
                 for package in repo_packages:
-                    check = f'rpm -q --whatprovides {package}'
+                    check = f'dnf whatprovides {package}'
                     if skip:
                         try:
                             guest.execute(f"{check}")
```

Comment:
What about `guest.execute(check)`? There's no need here to format `check` which is already a string under `f"..."`.